styles/main.scss

```diff
@@ -101,6 +108,228 @@ $font-family-secondary: "adelle", "Aleo", "serif";
   }
 }
 
+.home-hero {
+  z-index: -1;
+  width: 100%;
+  height: 600px;
+  object-fit: cover;
+  object-position: 40% 10%;
+}
+
+.home-hero-text-group {
+  bottom: 104px;
+  max-width: 1128px;
+  @media screen and (min-width: 1600px) {
```

Comment:
hmmm not really part of this card, but remind me (or whoever else whats to help tidy up the scss) to swap all this out to use `@include media-breakpoint-up/down/between/only` and specifying a size (sm/md/lg/xl/xxl) instead.
Using breakpoint sizes (https://getbootstrap.com/docs/5.2/layout/breakpoints/) means we just need to set whatever max-width is meant to be once.
I think for any future changes we should all remember to be consistent with stuff like this or call it out so we can decide on what the standard should be